manifests/agent.pp

```diff
@@ -34,6 +34,16 @@
       require => Package['logentries-daemon'],
       notify  => Service['logentries'],
     }
+  } elsif ( $logentries::params::agent_key == '' ) and ( $logentries::params::register == false ) {
```

Comment:
Register is a new parameter used for this change. It's set to true by
default so it does not interfere with previous behaviour. Datahub wasn't
tested, but should behave the same if you do not explicitly set register =
true.

On 29 April 2016 at 17:31, Gearoid O Fearghail <notifications@github.com>
wrote:

> In manifests/agent.pp
> <https://github.com/logentries/puppet-logentries/pull/4#discussion_r61596228>
> :
>
> > @@ -34,6 +34,16 @@
> >        require => Package['logentries-daemon'],
> >        notify  => Service['logentries'],
> >      }
> > +  } elsif ( $logentries::params::agent_key == '' ) and ( $logentries::params::register == false ) {
>
> Will this interfere with cases where Datahub is used?
>
> In those cases, the agent_key would be empty and registration would also
> be false.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/logentries/puppet-logentries/pull/4/files/5800aeb7b7a94406fac65954b45cee930c06eedf#r61596228>
>